trainingMaquetado/Maquetado.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+   
+	<div class= "headline"></div>
```

Comment:
no structure elements should be placed in the head tag. this one can be removed since you'll add the line to the upper-container div